docs/app/index.html

```diff
@@ -4,7 +4,7 @@
   <title>Stardust Docs</title>
 
   <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.4/semantic.min.css" />
-  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/androidstudio.min.css">
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/github.min.css">
```

Comment:
The dark theme was pretty loud on the light doc site, switched it to github theme:

![image](https://cloud.githubusercontent.com/assets/5067638/10531789/c9a1aeda-736a-11e5-95d0-2bf22fc24356.png)